kong/plugins/perimeterx/schema.lua

```diff
@@ -55,11 +55,7 @@ return {
             end
             return -1
         end
-        if config.custom_block_url ~= nil then -- verify custom_block_url is in uri_full
-            if config.whitelist == nil or config.whitelist.uri_full == nil or array_index_of(config.whitelist.uri_full, config.custom_block_url) == -1 then
```

Comment:
you can delete the function definition of `array_index_of` as no one uses it anymore